### PR TITLE
[4.0] Select Modules page

### PR DESCRIPTION
**Pull request by tecpromotion**
Pull Request für Issue #969 .

### Zusammenfassung der Änderungen

changes done. still open is #804 in general

### Wo wird der Sprachstring angezeigt / Wie kann getestet werden

backend

### CODE REVIEW

**Comment by tecpromotion** [comment]
@zero-24 after merge i can change open tasks to related issue #804 for this file

File: administrator/language/de-DE/de-DE.com_modules.ini [hunk 1]
```diff
@@ -194,13 +198,9 @@ COM_MODULES_POSITION_USER6="Benutzer 6"
 COM_MODULES_POSITION_USER7="Benutzer 7"
 COM_MODULES_POSITION_USER8="Benutzer 8"
 COM_MODULES_SAVE_SUCCESS="Das Modul wurde gespeichert!"
+COM_MODULES_SELECT_MODULE="Select module, %s"
```
Comment (zero-24) [inline administrator/language/de-DE/de-DE.com_modules.ini]: Translation missing @tecpromotion

File: administrator/language/de-DE/de-DE.com_modules.ini [hunk 2]
```diff
@@ -7,33 +7,39 @@
...
+COM_MODULES_ERR_XML="Modul-XML-Daten nicht verfügbar"
+COM_MODULES_EXPAND="Ausklappen"
 COM_MODULES_EXTENSION_PUBLISHED_DISABLED="Modul deaktiviert und veröffentlicht"
 COM_MODULES_EXTENSION_PUBLISHED_ENABLED="Modul aktiviert und veröffentlicht"
 COM_MODULES_EXTENSION_UNPUBLISHED_DISABLED="Modul deaktiviert und versteckt"
 COM_MODULES_EXTENSION_UNPUBLISHED_ENABLED="Modul aktiviert und versteckt"
 COM_MODULES_EXTRA_STYLE_DESC="Den Template-Stil für das Modul oder die Module in der ausgewählten Position überschreiben."
 COM_MODULES_EXTRA_STYLE_TITLE="Modulstil"
+COM_MODULES_FIELDSET_RULES="Modulberechtigungen"
+COM_MODULES_FIELD_BOOTSTRAP_SIZE_LABEL="Bootstrap-Größe"
```
Comment (zero-24) [inline administrator/language/de-DE/de-DE.com_modules.ini]: do we use that at other places too?

File: administrator/language/de-DE/de-DE.com_modules.ini (hunk 1 above)
Comment (tecpromotion) [reply administrator/language/de-DE/de-DE.com_modules.ini]: comes with issue #804

File: administrator/language/de-DE/de-DE.com_modules.ini (hunk 2 above)
Comment (tecpromotion) [reply administrator/language/de-DE/de-DE.com_modules.ini]: just reorder... no new translation added

**Comment by zero-24** [comment]
Thanks merging.

**Pull request merged**